Feature/mv unused dev

XianwuXue-NOAA opened the pull request:
Summarize of the changes:

1) Move unused module files, parm/gefs_awips, scripts, ush, util and sorc files to rocoto/bun/gefs_dev
2) Update and cleanup the pyGEFS to only keep the used tasks/programs
3) Move j-job, ex-script, gempak/fix[ush] scripts of task avg_gempak_vgf to rocoto/bun/gefs_dev
4) Remove the 'goto' in sorc/gefs_ensstat.fd/EPDF.f
5) Remove 'COMINgfs' in jobs/JGEFS_ATMOS_GEMPAK_META
6) Add more variables to rocoto/bin/wcoss_dell_p35/common.sh
7) Improve j-job and ush script of postsnd task
8) Improve ush script of prdgen task

XianwuXue-NOAA committed:
Move unused parm/gefs_awips, scripts, ush, util and sorc files to rocoto/bin/gefs_dev

 On branch feature/mv_unused_dev
	renamed:    parm/gefs_awips/grib2_naefs_gefs.pgrb2a_apcp -> rocoto/bin/gefs_dev/parm/gefs_awips/grib2_naefs_gefs.pgrb2a_apcp
	renamed:    parm/gefs_awips/grib2_naefs_gefs.pgrb2a_bc_tmp_2m -> rocoto/bin/gefs_dev/parm/gefs_awips/grib2_naefs_gefs.pgrb2a_bc_tmp_2m
	renamed:    parm/gefs_awips/grib_wafs37.ens00 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens00
	renamed:    parm/gefs_awips/grib_wafs37.ens06 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens06
	renamed:    parm/gefs_awips/grib_wafs37.ens102 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens102
	renamed:    parm/gefs_awips/grib_wafs37.ens108 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens108
	renamed:    parm/gefs_awips/grib_wafs37.ens114 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens114
	renamed:    parm/gefs_awips/grib_wafs37.ens12 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens12
	renamed:    parm/gefs_awips/grib_wafs37.ens120 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens120
	renamed:    parm/gefs_awips/grib_wafs37.ens126 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens126
	renamed:    parm/gefs_awips/grib_wafs37.ens132 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens132
	renamed:    parm/gefs_awips/grib_wafs37.ens138 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens138
	renamed:    parm/gefs_awips/grib_wafs37.ens144 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens144
	renamed:    parm/gefs_awips/grib_wafs37.ens150 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens150
	renamed:    parm/gefs_awips/grib_wafs37.ens156 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens156
	renamed:    parm/gefs_awips/grib_wafs37.ens162 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens162
	renamed:    parm/gefs_awips/grib_wafs37.ens168 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens168
	renamed:    parm/gefs_awips/grib_wafs37.ens174 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens174
	renamed:    parm/gefs_awips/grib_wafs37.ens18 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens18
	renamed:    parm/gefs_awips/grib_wafs37.ens180 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens180
	renamed:    parm/gefs_awips/grib_wafs37.ens186 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens186
	renamed:    parm/gefs_awips/grib_wafs37.ens192 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens192
	renamed:    parm/gefs_awips/grib_wafs37.ens24 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens24
	renamed:    parm/gefs_awips/grib_wafs37.ens30 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens30
	renamed:    parm/gefs_awips/grib_wafs37.ens36 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens36
	renamed:    parm/gefs_awips/grib_wafs37.ens42 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens42
	renamed:    parm/gefs_awips/grib_wafs37.ens48 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens48
	renamed:    parm/gefs_awips/grib_wafs37.ens54 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens54
	renamed:    parm/gefs_awips/grib_wafs37.ens60 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens60
	renamed:    parm/gefs_awips/grib_wafs37.ens66 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens66
	renamed:    parm/gefs_awips/grib_wafs37.ens72 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens72
	renamed:    parm/gefs_awips/grib_wafs37.ens78 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens78
	renamed:    parm/gefs_awips/grib_wafs37.ens84 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens84
	renamed:    parm/gefs_awips/grib_wafs37.ens90 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens90
	renamed:    parm/gefs_awips/grib_wafs37.ens96 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs37.ens96
	renamed:    parm/gefs_awips/grib_wafs38.ens00 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens00
	renamed:    parm/gefs_awips/grib_wafs38.ens06 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens06
	renamed:    parm/gefs_awips/grib_wafs38.ens102 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens102
	renamed:    parm/gefs_awips/grib_wafs38.ens108 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens108
	renamed:    parm/gefs_awips/grib_wafs38.ens114 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens114
	renamed:    parm/gefs_awips/grib_wafs38.ens12 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens12
	renamed:    parm/gefs_awips/grib_wafs38.ens120 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens120
	renamed:    parm/gefs_awips/grib_wafs38.ens126 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens126
	renamed:    parm/gefs_awips/grib_wafs38.ens132 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens132
	renamed:    parm/gefs_awips/grib_wafs38.ens138 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens138
	renamed:    parm/gefs_awips/grib_wafs38.ens144 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens144
	renamed:    parm/gefs_awips/grib_wafs38.ens150 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens150
	renamed:    parm/gefs_awips/grib_wafs38.ens156 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens156
	renamed:    parm/gefs_awips/grib_wafs38.ens162 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens162
	renamed:    parm/gefs_awips/grib_wafs38.ens168 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens168
	renamed:    parm/gefs_awips/grib_wafs38.ens174 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens174
	renamed:    parm/gefs_awips/grib_wafs38.ens18 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens18
	renamed:    parm/gefs_awips/grib_wafs38.ens180 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens180
	renamed:    parm/gefs_awips/grib_wafs38.ens186 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens186
	renamed:    parm/gefs_awips/grib_wafs38.ens192 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens192
	renamed:    parm/gefs_awips/grib_wafs38.ens24 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens24
	renamed:    parm/gefs_awips/grib_wafs38.ens30 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens30
	renamed:    parm/gefs_awips/grib_wafs38.ens36 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens36
	renamed:    parm/gefs_awips/grib_wafs38.ens42 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens42
	renamed:    parm/gefs_awips/grib_wafs38.ens48 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens48
	renamed:    parm/gefs_awips/grib_wafs38.ens54 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens54
	renamed:    parm/gefs_awips/grib_wafs38.ens60 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens60
	renamed:    parm/gefs_awips/grib_wafs38.ens66 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens66
	renamed:    parm/gefs_awips/grib_wafs38.ens72 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens72
	renamed:    parm/gefs_awips/grib_wafs38.ens78 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens78
	renamed:    parm/gefs_awips/grib_wafs38.ens84 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens84
	renamed:    parm/gefs_awips/grib_wafs38.ens90 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens90
	renamed:    parm/gefs_awips/grib_wafs38.ens96 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs38.ens96
	renamed:    parm/gefs_awips/grib_wafs39.ens00 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens00
	renamed:    parm/gefs_awips/grib_wafs39.ens06 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens06
	renamed:    parm/gefs_awips/grib_wafs39.ens102 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens102
	renamed:    parm/gefs_awips/grib_wafs39.ens108 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens108
	renamed:    parm/gefs_awips/grib_wafs39.ens114 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens114
	renamed:    parm/gefs_awips/grib_wafs39.ens12 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens12
	renamed:    parm/gefs_awips/grib_wafs39.ens120 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens120
    renamed:    parm/gefs_awips/grib_wafs39.ens126 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens126
	renamed:    parm/gefs_awips/grib_wafs39.ens132 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens132
	renamed:    parm/gefs_awips/grib_wafs39.ens138 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens138
    renamed:    parm/gefs_awips/grib_wafs39.ens144 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens144
    renamed:    parm/gefs_awips/grib_wafs39.ens150 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens150
    renamed:    parm/gefs_awips/grib_wafs39.ens156 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens156
	renamed:    parm/gefs_awips/grib_wafs39.ens162 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens162
	renamed:    parm/gefs_awips/grib_wafs39.ens168 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens168
	renamed:    parm/gefs_awips/grib_wafs39.ens174 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens174
	renamed:    parm/gefs_awips/grib_wafs39.ens18 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens18
	renamed:    parm/gefs_awips/grib_wafs39.ens180 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens180
	renamed:    parm/gefs_awips/grib_wafs39.ens186 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens186
	renamed:    parm/gefs_awips/grib_wafs39.ens192 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens192
	renamed:    parm/gefs_awips/grib_wafs39.ens24 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens24
	renamed:    parm/gefs_awips/grib_wafs39.ens30 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens30
	renamed:    parm/gefs_awips/grib_wafs39.ens36 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens36
	renamed:    parm/gefs_awips/grib_wafs39.ens42 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens42
	renamed:    parm/gefs_awips/grib_wafs39.ens48 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens48
	renamed:    parm/gefs_awips/grib_wafs39.ens54 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens54
	renamed:    parm/gefs_awips/grib_wafs39.ens60 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens60
	renamed:    parm/gefs_awips/grib_wafs39.ens66 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens66
	renamed:    parm/gefs_awips/grib_wafs39.ens72 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens72
	renamed:    parm/gefs_awips/grib_wafs39.ens78 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens78
	renamed:    parm/gefs_awips/grib_wafs39.ens84 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens84
	renamed:    parm/gefs_awips/grib_wafs39.ens90 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens90
	renamed:    parm/gefs_awips/grib_wafs39.ens96 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs39.ens96
	renamed:    parm/gefs_awips/grib_wafs40.ens00 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens00
	renamed:    parm/gefs_awips/grib_wafs40.ens06 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens06
	renamed:    parm/gefs_awips/grib_wafs40.ens102 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens102
	renamed:    parm/gefs_awips/grib_wafs40.ens108 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens108
	renamed:    parm/gefs_awips/grib_wafs40.ens114 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens114
	renamed:    parm/gefs_awips/grib_wafs40.ens12 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens12
	renamed:    parm/gefs_awips/grib_wafs40.ens120 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens120
	renamed:    parm/gefs_awips/grib_wafs40.ens126 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens126
	renamed:    parm/gefs_awips/grib_wafs40.ens132 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens132
	renamed:    parm/gefs_awips/grib_wafs40.ens138 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens138
	renamed:    parm/gefs_awips/grib_wafs40.ens144 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens144
	renamed:    parm/gefs_awips/grib_wafs40.ens150 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens150
	renamed:    parm/gefs_awips/grib_wafs40.ens156 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens156
	renamed:    parm/gefs_awips/grib_wafs40.ens162 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens162
	renamed:    parm/gefs_awips/grib_wafs40.ens168 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens168
	renamed:    parm/gefs_awips/grib_wafs40.ens174 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens174
	renamed:    parm/gefs_awips/grib_wafs40.ens18 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens18
	renamed:    parm/gefs_awips/grib_wafs40.ens180 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens180
	renamed:    parm/gefs_awips/grib_wafs40.ens186 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens186
    renamed:    parm/gefs_awips/grib_wafs40.ens192 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens192
	renamed:    parm/gefs_awips/grib_wafs40.ens24 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens24
	renamed:    parm/gefs_awips/grib_wafs40.ens30 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens30
	renamed:    parm/gefs_awips/grib_wafs40.ens36 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens36
	renamed:    parm/gefs_awips/grib_wafs40.ens42 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens42
	renamed:    parm/gefs_awips/grib_wafs40.ens48 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens48
	renamed:    parm/gefs_awips/grib_wafs40.ens54 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens54
	renamed:    parm/gefs_awips/grib_wafs40.ens60 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens60
	renamed:    parm/gefs_awips/grib_wafs40.ens66 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens66
	renamed:    parm/gefs_awips/grib_wafs40.ens72 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens72
	renamed:    parm/gefs_awips/grib_wafs40.ens78 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens78
	renamed:    parm/gefs_awips/grib_wafs40.ens84 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens84
	renamed:    parm/gefs_awips/grib_wafs40.ens90 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens90
	renamed:    parm/gefs_awips/grib_wafs40.ens96 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs40.ens96
	renamed:    parm/gefs_awips/grib_wafs41.ens00 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens00
	renamed:    parm/gefs_awips/grib_wafs41.ens06 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens06
	renamed:    parm/gefs_awips/grib_wafs41.ens102 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens102
	renamed:    parm/gefs_awips/grib_wafs41.ens108 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens108
	renamed:    parm/gefs_awips/grib_wafs41.ens114 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens114
	renamed:    parm/gefs_awips/grib_wafs41.ens12 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens12
	renamed:    parm/gefs_awips/grib_wafs41.ens120 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens120
	renamed:    parm/gefs_awips/grib_wafs41.ens126 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens126
	renamed:    parm/gefs_awips/grib_wafs41.ens132 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens132
	renamed:    parm/gefs_awips/grib_wafs41.ens138 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens138
	renamed:    parm/gefs_awips/grib_wafs41.ens144 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens144
	renamed:    parm/gefs_awips/grib_wafs41.ens150 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens150
	renamed:    parm/gefs_awips/grib_wafs41.ens156 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens156
	renamed:    parm/gefs_awips/grib_wafs41.ens162 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens162
	renamed:    parm/gefs_awips/grib_wafs41.ens168 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens168
	renamed:    parm/gefs_awips/grib_wafs41.ens174 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens174
	renamed:    parm/gefs_awips/grib_wafs41.ens18 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens18
	renamed:    parm/gefs_awips/grib_wafs41.ens180 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens180
	renamed:    parm/gefs_awips/grib_wafs41.ens186 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens186
	renamed:    parm/gefs_awips/grib_wafs41.ens192 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens192
	renamed:    parm/gefs_awips/grib_wafs41.ens24 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens24
	renamed:    parm/gefs_awips/grib_wafs41.ens30 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens30
	renamed:    parm/gefs_awips/grib_wafs41.ens36 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens36
	renamed:    parm/gefs_awips/grib_wafs41.ens42 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens42
	renamed:    parm/gefs_awips/grib_wafs41.ens48 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens48
	renamed:    parm/gefs_awips/grib_wafs41.ens54 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens54
	renamed:    parm/gefs_awips/grib_wafs41.ens60 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens60
	renamed:    parm/gefs_awips/grib_wafs41.ens66 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens66
	renamed:    parm/gefs_awips/grib_wafs41.ens72 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens72
	renamed:    parm/gefs_awips/grib_wafs41.ens78 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens78
	renamed:    parm/gefs_awips/grib_wafs41.ens84 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens84
	renamed:    parm/gefs_awips/grib_wafs41.ens90 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens90
	renamed:    parm/gefs_awips/grib_wafs41.ens96 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs41.ens96
	renamed:    parm/gefs_awips/grib_wafs42.ens00 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens00
	renamed:    parm/gefs_awips/grib_wafs42.ens06 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens06
	renamed:    parm/gefs_awips/grib_wafs42.ens102 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens102
	renamed:    parm/gefs_awips/grib_wafs42.ens108 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens108
	renamed:    parm/gefs_awips/grib_wafs42.ens114 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens114
	renamed:    parm/gefs_awips/grib_wafs42.ens12 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens12
	renamed:    parm/gefs_awips/grib_wafs42.ens120 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens120
	renamed:    parm/gefs_awips/grib_wafs42.ens126 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens126
	renamed:    parm/gefs_awips/grib_wafs42.ens132 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens132
	renamed:    parm/gefs_awips/grib_wafs42.ens138 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens138
	renamed:    parm/gefs_awips/grib_wafs42.ens144 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens144
	renamed:    parm/gefs_awips/grib_wafs42.ens150 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens150
	renamed:    parm/gefs_awips/grib_wafs42.ens156 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens156
	renamed:    parm/gefs_awips/grib_wafs42.ens162 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens162
	renamed:    parm/gefs_awips/grib_wafs42.ens168 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens168
	renamed:    parm/gefs_awips/grib_wafs42.ens174 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens174
	renamed:    parm/gefs_awips/grib_wafs42.ens18 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens18
	renamed:    parm/gefs_awips/grib_wafs42.ens180 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens180
	renamed:    parm/gefs_awips/grib_wafs42.ens186 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens186
	renamed:    parm/gefs_awips/grib_wafs42.ens192 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens192
	renamed:    parm/gefs_awips/grib_wafs42.ens24 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens24
	renamed:    parm/gefs_awips/grib_wafs42.ens30 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens30
	renamed:    parm/gefs_awips/grib_wafs42.ens36 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens36
	renamed:    parm/gefs_awips/grib_wafs42.ens42 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens42
	renamed:    parm/gefs_awips/grib_wafs42.ens48 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens48
	renamed:    parm/gefs_awips/grib_wafs42.ens54 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens54
	renamed:    parm/gefs_awips/grib_wafs42.ens60 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens60
	renamed:    parm/gefs_awips/grib_wafs42.ens66 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens66
	renamed:    parm/gefs_awips/grib_wafs42.ens72 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens72
	renamed:    parm/gefs_awips/grib_wafs42.ens78 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens78
	renamed:    parm/gefs_awips/grib_wafs42.ens84 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens84
	renamed:    parm/gefs_awips/grib_wafs42.ens90 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens90
	renamed:    parm/gefs_awips/grib_wafs42.ens96 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs42.ens96
	renamed:    parm/gefs_awips/grib_wafs43.ens00 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens00
	renamed:    parm/gefs_awips/grib_wafs43.ens06 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens06
	renamed:    parm/gefs_awips/grib_wafs43.ens102 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens102
	renamed:    parm/gefs_awips/grib_wafs43.ens108 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens108
	renamed:    parm/gefs_awips/grib_wafs43.ens114 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens114
	renamed:    parm/gefs_awips/grib_wafs43.ens12 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens12
	renamed:    parm/gefs_awips/grib_wafs43.ens120 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens120
	renamed:    parm/gefs_awips/grib_wafs43.ens126 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens126
	renamed:    parm/gefs_awips/grib_wafs43.ens132 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens132
	renamed:    parm/gefs_awips/grib_wafs43.ens138 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens138
	renamed:    parm/gefs_awips/grib_wafs43.ens144 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens144
	renamed:    parm/gefs_awips/grib_wafs43.ens150 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens150
	renamed:    parm/gefs_awips/grib_wafs43.ens156 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens156
	renamed:    parm/gefs_awips/grib_wafs43.ens162 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens162
	renamed:    parm/gefs_awips/grib_wafs43.ens168 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens168
	renamed:    parm/gefs_awips/grib_wafs43.ens174 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens174
	renamed:    parm/gefs_awips/grib_wafs43.ens18 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens18
	renamed:    parm/gefs_awips/grib_wafs43.ens180 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens180
	renamed:    parm/gefs_awips/grib_wafs43.ens186 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens186
	renamed:    parm/gefs_awips/grib_wafs43.ens192 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens192
	renamed:    parm/gefs_awips/grib_wafs43.ens24 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens24
	renamed:    parm/gefs_awips/grib_wafs43.ens30 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens30
	renamed:    parm/gefs_awips/grib_wafs43.ens36 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens36
	renamed:    parm/gefs_awips/grib_wafs43.ens42 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens42
	renamed:    parm/gefs_awips/grib_wafs43.ens48 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens48
	renamed:    parm/gefs_awips/grib_wafs43.ens54 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens54
	renamed:    parm/gefs_awips/grib_wafs43.ens60 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens60
	renamed:    parm/gefs_awips/grib_wafs43.ens66 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens66
	renamed:    parm/gefs_awips/grib_wafs43.ens72 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens72
	renamed:    parm/gefs_awips/grib_wafs43.ens78 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens78
	renamed:    parm/gefs_awips/grib_wafs43.ens84 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens84
	renamed:    parm/gefs_awips/grib_wafs43.ens90 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens90
	renamed:    parm/gefs_awips/grib_wafs43.ens96 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs43.ens96
	renamed:    parm/gefs_awips/grib_wafs44.ens00 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens00
	renamed:    parm/gefs_awips/grib_wafs44.ens06 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens06
	renamed:    parm/gefs_awips/grib_wafs44.ens102 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens102
	renamed:    parm/gefs_awips/grib_wafs44.ens108 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens108
	renamed:    parm/gefs_awips/grib_wafs44.ens114 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens114
	renamed:    parm/gefs_awips/grib_wafs44.ens12 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens12
	renamed:    parm/gefs_awips/grib_wafs44.ens120 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens120
	renamed:    parm/gefs_awips/grib_wafs44.ens126 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens126
	renamed:    parm/gefs_awips/grib_wafs44.ens132 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens132
	renamed:    parm/gefs_awips/grib_wafs44.ens138 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens138
	renamed:    parm/gefs_awips/grib_wafs44.ens144 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens144
	renamed:    parm/gefs_awips/grib_wafs44.ens150 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens150
	renamed:    parm/gefs_awips/grib_wafs44.ens156 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens156
	renamed:    parm/gefs_awips/grib_wafs44.ens162 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens162
	renamed:    parm/gefs_awips/grib_wafs44.ens168 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens168
	renamed:    parm/gefs_awips/grib_wafs44.ens174 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens174
	renamed:    parm/gefs_awips/grib_wafs44.ens18 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens18
	renamed:    parm/gefs_awips/grib_wafs44.ens180 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens180
	renamed:    parm/gefs_awips/grib_wafs44.ens186 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens186
	renamed:    parm/gefs_awips/grib_wafs44.ens192 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens192
	renamed:    parm/gefs_awips/grib_wafs44.ens24 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens24
	renamed:    parm/gefs_awips/grib_wafs44.ens30 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens30
	renamed:    parm/gefs_awips/grib_wafs44.ens36 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens36
	renamed:    parm/gefs_awips/grib_wafs44.ens42 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens42
	renamed:    parm/gefs_awips/grib_wafs44.ens48 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens48
	renamed:    parm/gefs_awips/grib_wafs44.ens54 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens54
	renamed:    parm/gefs_awips/grib_wafs44.ens60 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens60
	renamed:    parm/gefs_awips/grib_wafs44.ens66 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens66
	renamed:    parm/gefs_awips/grib_wafs44.ens72 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens72
	renamed:    parm/gefs_awips/grib_wafs44.ens78 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens78
	renamed:    parm/gefs_awips/grib_wafs44.ens84 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens84
	renamed:    parm/gefs_awips/grib_wafs44.ens90 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens90
	renamed:    parm/gefs_awips/grib_wafs44.ens96 -> rocoto/bin/gefs_dev/parm/gefs_awips/grib_wafs44.ens96
	renamed:    scripts/exgefs_extractvars.sh.ecf -> rocoto/bin/gefs_dev/scripts/exgefs_extractvars.sh.ecf
	renamed:    scripts/exgefs_rf_prep.sh.ecf -> rocoto/bin/gefs_dev/scripts/exgefs_rf_prep.sh.ecf
	renamed:    scripts/exgefs_wafs.sh.ecf -> rocoto/bin/gefs_dev/scripts/exgefs_wafs.sh.ecf
	renamed:    scripts/exglobal_enscqpf.sh.ecf -> rocoto/bin/gefs_dev/scripts/exglobal_enscqpf.sh.ecf
	renamed:    sorc/build_global_enscqpf.sh -> rocoto/bin/gefs_dev/sorc/build_global_enscqpf.sh
	renamed:    sorc/build_global_enscvprcp.sh -> rocoto/bin/gefs_dev/sorc/build_global_enscvprcp.sh
	renamed:    sorc/build_global_enscvt24h.sh -> rocoto/bin/gefs_dev/sorc/build_global_enscvt24h.sh
	renamed:    sorc/build_global_enspqpf.sh -> rocoto/bin/gefs_dev/sorc/build_global_enspqpf.sh
	renamed:    sorc/build_global_enspvrfy.sh -> rocoto/bin/gefs_dev/sorc/build_global_enspvrfy.sh
	renamed:    sorc/build_global_ensrfmat.sh -> rocoto/bin/gefs_dev/sorc/build_global_ensrfmat.sh
	renamed:    sorc/build_global_enssrbias.sh -> rocoto/bin/gefs_dev/sorc/build_global_enssrbias.sh
	renamed:    sorc/build_overenstr_grib.sh -> rocoto/bin/gefs_dev/sorc/build_overenstr_grib.sh
	renamed:    sorc/global_enscqpf.fd/CQPF.f90 -> rocoto/bin/gefs_dev/sorc/global_enscqpf.fd/CQPF.f90
	renamed:    sorc/global_enscqpf.fd/CQPF.f90_22m -> rocoto/bin/gefs_dev/sorc/global_enscqpf.fd/CQPF.f90_22m
	renamed:    sorc/global_enscqpf.fd/INIT_PARM.f90 -> rocoto/bin/gefs_dev/sorc/global_enscqpf.fd/INIT_PARM.f90
	renamed:    sorc/global_enscqpf.fd/Makefile -> rocoto/bin/gefs_dev/sorc/global_enscqpf.fd/Makefile
	renamed:    sorc/global_enscqpf.fd/PRINTINFR.f90 -> rocoto/bin/gefs_dev/sorc/global_enscqpf.fd/PRINTINFR.f90
	renamed:    sorc/global_enscqpf.fd/QTPINT.f90 -> rocoto/bin/gefs_dev/sorc/global_enscqpf.fd/QTPINT.f90
	renamed:    sorc/global_enscvprcp.fd/CVPRCP.f90 -> rocoto/bin/gefs_dev/sorc/global_enscvprcp.fd/CVPRCP.f90
	renamed:    sorc/global_enscvprcp.fd/CVPRCP.f90_22m -> rocoto/bin/gefs_dev/sorc/global_enscvprcp.fd/CVPRCP.f90_22m
	renamed:    sorc/global_enscvprcp.fd/CVPRCP.f90_22m_newdata -> rocoto/bin/gefs_dev/sorc/global_enscvprcp.fd/CVPRCP.f90_22m_newdata
	renamed:    sorc/global_enscvprcp.fd/INIT_PARM.f90 -> rocoto/bin/gefs_dev/sorc/global_enscvprcp.fd/INIT_PARM.f90
	renamed:    sorc/global_enscvprcp.fd/Makefile -> rocoto/bin/gefs_dev/sorc/global_enscvprcp.fd/Makefile
	renamed:    sorc/global_enscvprcp.fd/PRINTINFR.f90 -> rocoto/bin/gefs_dev/sorc/global_enscvprcp.fd/PRINTINFR.f90
	renamed:    sorc/global_enscvt24h.fd/CVT24H.f90 -> rocoto/bin/gefs_dev/sorc/global_enscvt24h.fd/CVT24H.f90
	renamed:    sorc/global_enscvt24h.fd/CVT24H.f90_22m -> rocoto/bin/gefs_dev/sorc/global_enscvt24h.fd/CVT24H.f90_22m
	renamed:    sorc/global_enscvt24h.fd/INIT_PARM.f90 -> rocoto/bin/gefs_dev/sorc/global_enscvt24h.fd/INIT_PARM.f90
	renamed:    sorc/global_enscvt24h.fd/Makefile -> rocoto/bin/gefs_dev/sorc/global_enscvt24h.fd/Makefile
	renamed:    sorc/global_enscvt24h.fd/PRINTINFR.f90 -> rocoto/bin/gefs_dev/sorc/global_enscvt24h.fd/PRINTINFR.f90
	renamed:    sorc/global_enspqpf.fd/Makefile -> rocoto/bin/gefs_dev/sorc/global_enspqpf.fd/Makefile
	renamed:    sorc/global_enspqpf.fd/PQPF.f90 -> rocoto/bin/gefs_dev/sorc/global_enspqpf.fd/PQPF.f90
	renamed:    sorc/global_enspqpf.fd/init_parm.f90 -> rocoto/bin/gefs_dev/sorc/global_enspqpf.fd/init_parm.f90
	renamed:    sorc/global_enspqpf.fd/printinfr.f90 -> rocoto/bin/gefs_dev/sorc/global_enspqpf.fd/printinfr.f90
	renamed:    sorc/global_enspvrfy.fd/ANNATV.f90 -> rocoto/bin/gefs_dev/sorc/global_enspvrfy.fd/ANNATV.f90
	renamed:    sorc/global_enspvrfy.fd/INIT_PARM.f90 -> rocoto/bin/gefs_dev/sorc/global_enspvrfy.fd/INIT_PARM.f90
	renamed:    sorc/global_enspvrfy.fd/Makefile -> rocoto/bin/gefs_dev/sorc/global_enspvrfy.fd/Makefile
	renamed:    sorc/global_enspvrfy.fd/PRINTINFR.f90 -> rocoto/bin/gefs_dev/sorc/global_enspvrfy.fd/PRINTINFR.f90
	renamed:    sorc/global_enspvrfy.fd/PVRFY.f90 -> rocoto/bin/gefs_dev/sorc/global_enspvrfy.fd/PVRFY.f90
	renamed:    sorc/global_enspvrfy.fd/PVRFY.f90_22m -> rocoto/bin/gefs_dev/sorc/global_enspvrfy.fd/PVRFY.f90_22m
	renamed:    sorc/global_enspvrfy.fd/VRFY.f90 -> rocoto/bin/gefs_dev/sorc/global_enspvrfy.fd/VRFY.f90
	renamed:    sorc/global_ensrfmat.fd/Makefile -> rocoto/bin/gefs_dev/sorc/global_ensrfmat.fd/Makefile
	renamed:    sorc/global_ensrfmat.fd/REFORMAT.f -> rocoto/bin/gefs_dev/sorc/global_ensrfmat.fd/REFORMAT.f
	renamed:    sorc/global_ensrfmat.fd/REFORMAT.f_ccs -> rocoto/bin/gefs_dev/sorc/global_ensrfmat.fd/REFORMAT.f_ccs
	renamed:    sorc/global_enssrbias.fd/Makefile -> rocoto/bin/gefs_dev/sorc/global_enssrbias.fd/Makefile
	renamed:    sorc/global_enssrbias.fd/STAT_RM_BIAS_30D.f -> rocoto/bin/gefs_dev/sorc/global_enssrbias.fd/STAT_RM_BIAS_30D.f
	renamed:    ush/gefs_exishef.sh -> rocoto/bin/gefs_dev/ush/gefs_exishef.sh
	renamed:    ush/gefs_mmefs_awips.sh -> rocoto/bin/gefs_dev/ush/gefs_mmefs_awips.sh
	renamed:    ush/global_enscvprcp.sh -> rocoto/bin/gefs_dev/ush/global_enscvprcp.sh
	renamed:    ush/global_enscvt24h.sh -> rocoto/bin/gefs_dev/ush/global_enscvt24h.sh
	renamed:    ush/global_enspqpf_24h_cmc.sh -> rocoto/bin/gefs_dev/ush/global_enspqpf_24h_cmc.sh
	renamed:    ush/global_ensprcpcv.sh -> rocoto/bin/gefs_dev/ush/global_ensprcpcv.sh
	renamed:    ush/global_enspvrfy.sh -> rocoto/bin/gefs_dev/ush/global_enspvrfy.sh
	renamed:    ush/global_ensrfmat.sh -> rocoto/bin/gefs_dev/ush/global_ensrfmat.sh
	renamed:    ush/global_enssrbias.sh -> rocoto/bin/gefs_dev/ush/global_enssrbias.sh
	renamed:    ush/global_enswgrp.sh -> rocoto/bin/gefs_dev/ush/global_enswgrp.sh
	renamed:    ush/mkgefs_wafs.sh -> rocoto/bin/gefs_dev/ush/mkgefs_wafs.sh
	renamed:    ush/syndataxtrack.sh -> rocoto/bin/gefs_dev/ush/syndataxtrack.sh
	renamed:    util/sorc/overenstr.grib.fd/Makefile -> rocoto/bin/gefs_dev/util/sorc/overenstr.grib.fd/Makefile
	renamed:    util/sorc/overenstr.grib.fd/buld_overenstr.sh -> rocoto/bin/gefs_dev/util/sorc/overenstr.grib.fd/buld_overenstr.sh
	renamed:    util/sorc/overenstr.grib.fd/overenstr.grib.f -> rocoto/bin/gefs_dev/util/sorc/overenstr.grib.fd/overenstr.grib.f
	renamed:    util/sorc/overenstr.grib.fd/sample.script -> rocoto/bin/gefs_dev/util/sorc/overenstr.grib.fd/sample.script
	renamed:    util/ush/extrkr.sh -> rocoto/bin/gefs_dev/util/ush/extrkr.sh
	renamed:    util/ush/ingest_delete-bad-dates.pl -> rocoto/bin/gefs_dev/util/ush/ingest_delete-bad-dates.pl
	renamed:    util/ush/ingest_uniq-final.awk -> rocoto/bin/gefs_dev/util/ush/ingest_uniq-final.awk

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Move unused modulefiles to rocoto/bin/gefs_dev/modulefiles/gefs

 On branch feature/mv_unused_dev
 Changes to be committed:
	renamed:    ../modulefiles/gefs/global_enscqpf.hera -> bin/gefs_dev/modulefiles/gefs/global_enscqpf.hera
	renamed:    ../modulefiles/gefs/global_enscqpf.jet -> bin/gefs_dev/modulefiles/gefs/global_enscqpf.jet
	renamed:    ../modulefiles/gefs/global_enscqpf.wcoss_cray -> bin/gefs_dev/modulefiles/gefs/global_enscqpf.wcoss_cray
	renamed:    ../modulefiles/gefs/global_enscqpf.wcoss_dell_p3 -> bin/gefs_dev/modulefiles/gefs/global_enscqpf.wcoss_dell_p3
	renamed:    ../modulefiles/gefs/global_enscvprcp.hera -> bin/gefs_dev/modulefiles/gefs/global_enscvprcp.hera
	renamed:    ../modulefiles/gefs/global_enscvprcp.jet -> bin/gefs_dev/modulefiles/gefs/global_enscvprcp.jet
	renamed:    ../modulefiles/gefs/global_enscvprcp.wcoss_cray -> bin/gefs_dev/modulefiles/gefs/global_enscvprcp.wcoss_cray
	renamed:    ../modulefiles/gefs/global_enscvprcp.wcoss_dell_p3 -> bin/gefs_dev/modulefiles/gefs/global_enscvprcp.wcoss_dell_p3
	renamed:    ../modulefiles/gefs/global_enscvt24h.hera -> bin/gefs_dev/modulefiles/gefs/global_enscvt24h.hera
	renamed:    ../modulefiles/gefs/global_enscvt24h.jet -> bin/gefs_dev/modulefiles/gefs/global_enscvt24h.jet
	renamed:    ../modulefiles/gefs/global_enscvt24h.wcoss_cray -> bin/gefs_dev/modulefiles/gefs/global_enscvt24h.wcoss_cray
	renamed:    ../modulefiles/gefs/global_enscvt24h.wcoss_dell_p3 -> bin/gefs_dev/modulefiles/gefs/global_enscvt24h.wcoss_dell_p3
	renamed:    ../modulefiles/gefs/global_enspqpf.hera -> bin/gefs_dev/modulefiles/gefs/global_enspqpf.hera
	renamed:    ../modulefiles/gefs/global_enspqpf.jet -> bin/gefs_dev/modulefiles/gefs/global_enspqpf.jet
	renamed:    ../modulefiles/gefs/global_enspqpf.wcoss_cray -> bin/gefs_dev/modulefiles/gefs/global_enspqpf.wcoss_cray
	renamed:    ../modulefiles/gefs/global_enspqpf.wcoss_dell_p3 -> bin/gefs_dev/modulefiles/gefs/global_enspqpf.wcoss_dell_p3
	renamed:    ../modulefiles/gefs/global_enspvrfy.hera -> bin/gefs_dev/modulefiles/gefs/global_enspvrfy.hera
	renamed:    ../modulefiles/gefs/global_enspvrfy.jet -> bin/gefs_dev/modulefiles/gefs/global_enspvrfy.jet
	renamed:    ../modulefiles/gefs/global_enspvrfy.wcoss_cray -> bin/gefs_dev/modulefiles/gefs/global_enspvrfy.wcoss_cray
	renamed:    ../modulefiles/gefs/global_enspvrfy.wcoss_dell_p3 -> bin/gefs_dev/modulefiles/gefs/global_enspvrfy.wcoss_dell_p3
	renamed:    ../modulefiles/gefs/global_ensrfmat.hera -> bin/gefs_dev/modulefiles/gefs/global_ensrfmat.hera
	renamed:    ../modulefiles/gefs/global_ensrfmat.jet -> bin/gefs_dev/modulefiles/gefs/global_ensrfmat.jet
	renamed:    ../modulefiles/gefs/global_ensrfmat.wcoss_cray -> bin/gefs_dev/modulefiles/gefs/global_ensrfmat.wcoss_cray
	renamed:    ../modulefiles/gefs/global_ensrfmat.wcoss_dell_p3 -> bin/gefs_dev/modulefiles/gefs/global_ensrfmat.wcoss_dell_p3
	renamed:    ../modulefiles/gefs/global_enssrbias.hera -> bin/gefs_dev/modulefiles/gefs/global_enssrbias.hera
	renamed:    ../modulefiles/gefs/global_enssrbias.jet -> bin/gefs_dev/modulefiles/gefs/global_enssrbias.jet
	renamed:    ../modulefiles/gefs/global_enssrbias.wcoss_cray -> bin/gefs_dev/modulefiles/gefs/global_enssrbias.wcoss_cray
	renamed:    ../modulefiles/gefs/global_enssrbias.wcoss_dell_p3 -> bin/gefs_dev/modulefiles/gefs/global_enssrbias.wcoss_dell_p3
	renamed:    ../modulefiles/gefs/overenstr.grib.hera -> bin/gefs_dev/modulefiles/gefs/overenstr.grib.hera
	renamed:    ../modulefiles/gefs/overenstr.grib.jet -> bin/gefs_dev/modulefiles/gefs/overenstr.grib.jet
	renamed:    ../modulefiles/gefs/overenstr.grib.wcoss_cray -> bin/gefs_dev/modulefiles/gefs/overenstr.grib.wcoss_cray
	renamed:    ../modulefiles/gefs/overenstr.grib.wcoss_dell_p3 -> bin/gefs_dev/modulefiles/gefs/overenstr.grib.wcoss_dell_p3

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Update the build_all, gefs_build.cfg and partial_build.sh in sorc
    to cleanup the unused programs
Copy build_all, gefs_build.cfg and partial_build.sh to bin/gefs_dev/sorc
    and change the name to build_all_dev.sh, gefs_build_dev.cfg and partial_build_dev.sh
    to build for dev programs
Improve compile_install_all.sh to make clean for the existing folders

 On branch feature/mv_unused_dev
	new file:   bin/gefs_dev/sorc/build_all_dev.sh
	new file:   bin/gefs_dev/sorc/gefs_build_dev.cfg
	new file:   bin/gefs_dev/sorc/partial_build_dev.sh
	modified:   compile_install_all.sh
	modified:   ../sorc/build_all.sh
	modified:   ../sorc/gefs_build.cfg
	modified:   ../sorc/partial_build.sh

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Move j-job, exscript and gempak/fix & use file of task ATMOS_AVG_GEMPAK_VGF to rocoto/bin/gefs_dev/gempak/
 On branch feature/mv_unused_dev
 Changes to be committed:
	renamed:    ../gempak/fix/g2varswmo2.tbl.orig0827 -> bin/gefs_dev/gempak/fix/g2varswmo2.tbl.orig0827
	renamed:    ../gempak/fix/g2varswmo2.tbl.test -> bin/gefs_dev/gempak/fix/g2varswmo2.tbl.test
	renamed:    ../gempak/fix/g2varswmo2.tbl.test2 -> bin/gefs_dev/gempak/fix/g2varswmo2.tbl.test2
	renamed:    ../gempak/fix/gefs_avg_vgf_00 -> bin/gefs_dev/gempak/fix/gefs_avg_vgf_00
	renamed:    ../gempak/fix/gefs_avg_vgf_12 -> bin/gefs_dev/gempak/fix/gefs_avg_vgf_12
	renamed:    ../gempak/ush/gefs_avg_gempak_vgf.sh -> bin/gefs_dev/gempak/ush/gefs_avg_gempak_vgf.sh
	renamed:    ../jobs/JGEFS_ATMOS_AVG_GEMPAK_VGF -> bin/gefs_dev/jobs/JGEFS_ATMOS_AVG_GEMPAK_VGF
	renamed:    ../scripts/exgefs_avg_gempak_vgf.sh.ecf -> bin/gefs_dev/scripts/exgefs_avg_gempak_vgf.sh.ecf

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Remove avgspr_gempak_vgf task from pyGEFS

 On branch feature/mv_unused_dev
	modified:   py/GEFS_XML_For_Tasks.py

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
1) Remove the 'goto' in sorc/gefs_ensstat.fd/EPDF.f
2) Remove 'COMINgfs' in jobs/JGEFS_ATMOS_GEMPAK_META
3) Add more variables to rocoto/bin/wcoss_dell_p35/common.sh
 On branch feature/mv_unused_dev
 Changes to be committed:
	modified:   ../jobs/JGEFS_ATMOS_GEMPAK_META
	modified:   bin/wcoss_dell_p35/common.sh
	modified:   ../sorc/gefs_ensstat.fd/EPDF.f

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
1) Improve j-job and ush script of postsnd task
2) Improve ush script of prdgen task

 On branch feature/ee_updates
 Changes to be committed:
	modified:   jobs/JGEFS_ATMOS_POSTSND
	modified:   ush/gefs_prdgen.sh
	modified:   ush/gefs_sndp.sh

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Merge branch 'feature/ee_updates' into feature/mv_unused_dev

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Merge remote-tracking branch 'upstream/master' into feature/mv_unused_dev

WalterKolczynski-NOAA commented on sorc/gefs_ensstat.fd/EPDF.f:
```diff
@@ -240,10 +240,10 @@ function epdf(ef,ew,n,qq,ictl)
        if (qq.lt.q(i)) then
         k=i-1
-        goto 101
+        exit
```
This is functionally different than the control. Are we sure this performs as desired?

WalterKolczynski-NOAA replied:
Nevermind. Not used to dealing with FORTRAN loops.

WalterKolczynski-NOAA reviewed (commented):
See comments

WalterKolczynski-NOAA commented on ush/gefs_sndp.sh:
```diff
@@ -61,7 +61,7 @@ EOF
     if [ $SENDCOM = 'YES' ]
     then 
-      if [ $SENDDBN = 'YES' ] ; then
+      if [ $SENDDBN_NTC = 'YES' ] ; then
```
Why the change? It seems counter to the request to change from SENDDBN_GB2. Is it confirmed that NCO approves of this change?

XianwuXue-NOAA replied:
Add Dingchen to the reviewer, This is what he changed.

WalterKolczynski-NOAA replied:
Rereading the Implementation Standards, this seems correct. But I am going to double-check with Steven in the morning.

WalterKolczynski-NOAA reviewed (commented):
One more comment

XianwuXue-NOAA committed:
Replace "init_fv3chgrs" to "prep" to satisfy NCO's requrements
 On branch feature/mv_unused_dev
 Changes to be committed:
	modified:   bin/cray/init_fv3chgrs.sh
	modified:   bin/hera/init_fv3chgrs.sh
	modified:   bin/wcoss_dell_p3/init_fv3chgrs.sh
	modified:   bin/wcoss_dell_p35/common.sh
	modified:   bin/wcoss_dell_p35/init_fv3chgrs.sh
	modified:   py/GEFS_XML_For_Tasks.py
	modified:   py/job_id.conf
	modified:   py/user_cray.conf
	modified:   py/user_hera.conf
	modified:   py/user_wcoss_dell_p3.conf
	modified:   py/user_wcoss_dell_p35.conf
	modified:   user_full.conf

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA committed:
Rename the j-job and bin files for init_fv3chgrs to prep

 On branch feature/mv_unused_dev
	renamed:    ../jobs/JGEFS_ATMOS_INIT_FV3CHGRS -> ../jobs/JGEFS_ATMOS_PREP
	renamed:    bin/cray/init_fv3chgrs.sh -> bin/cray/prep.sh
	renamed:    bin/hera/init_fv3chgrs.sh -> bin/hera/prep.sh
	renamed:    bin/wcoss_dell_p3/init_fv3chgrs.sh -> bin/wcoss_dell_p3/prep.sh
	renamed:    bin/wcoss_dell_p35/init_fv3chgrs.sh -> bin/wcoss_dell_p35/prep.sh

Refs: #76266, #64583, #62644, #71500 and #56936

XianwuXue-NOAA reviewed (commented):
Look like, I can not approve it. So I used the comment to approve it.
